frontend/src/components/templates/UserEnrollmentList/NoEnrollments.css

```diff
@@ -0,0 +1,12 @@
+.no-enrollments-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 20px;
+    padding: 80px 0;
+}
+
+.no-enrollments-container p {
+    margin: 0;
```

Comment:
this override the rule `gap: 20px;`

I'd rather use a one feat all rule instead of a generic rules used twice and overrided on is second and last usage.

Reply:
Nope, this rule doesn't override `gap` it overrides the default `margin` set to each `p` elements, which is : `var(--text-spacing)`.